src/Microsoft.AspNetCore.Razor/Tokenizer/Symbols/SymbolExtensions.cs

```diff
@@ -23,14 +37,21 @@ public static LocationTagged<string> GetContent(this SpanBuilder span, Func<IEnu
 
         public static LocationTagged<string> GetContent(this IEnumerable<ISymbol> symbols, SourceLocation spanStart)
```

Comment:
6 each (the first one gets used most from CSharpParser.Directives.cs, the latter from HtmlMarkupParser